src/scss/index.css

```diff
@@ -0,0 +1,134 @@
+body {
```

Comment:
I thought you are to use SCSS and then transpile it to CSS with SASS npm package. Why the switch?

Reply:
hmm, if you look at index.scss there you can see that is has compiled scss to css?